### ISSUE TITLE
Automatic miniature in the list of notes

### ISSUE THREAD

**Issue by zorglubu** [PhieF/CarnetNextcloud#19]
Within the browser displaying recent or all notes, it would be nice to have, near the title or just under it, a miniature created with the first image of the note.
It works with an added media, but not with an image inserted inside the note with a copy-paste.

**Comment by PhieF** [comment]
Yep because media added inside are not officially supported, yet haha
